README.md

```diff
@@ -52,15 +52,39 @@ We provide several notebooks to show how recommendation algorithms can be design
 
 - The [Operationalizion Notebook](notebooks/05_operationalize) demonstrates how to deploy models in production systems.
 
+
+The Quick-Start and Modeling notebooks showcase how to utilize the following algorithms to build a recommender system:
+
+**Algorithms**
+
+| Algorithm | Environment | Description | 
+| --- | --- | --- |
+| **`Classic Recommenders`** |
+| [Surprise/Singular Value Decomposition (SVD)](notebooks/00_quick_start/sar_single_node_movielens.ipynb) | Python |  | 
+| [Alternating Least Squares (ALS)](notebooks/00_quick_start/als_pyspark_movielens.ipynb) | Spark |  |
+| **`Microsoft Recommenders`** |
```

Comment:
One problem I see with the classification of classic, Microsoft and DL recommenders is that xDeepFM and DKN are also MS recommenders, but they are in DL...

Reply:
Moved xDeepFM and DKN to MS (still open to how to best show this)